From 08175208f7d567017250957fc2a9838636c527b5 Mon Sep 17 00:00:00 2001
From: Andrej Rode <mail@andrejro.de>
Date: Sun, 11 Feb 2018 14:21:28 +0100
Subject: [PATCH] add option to disable bash completion

---
 cmake/BashCompletion.cmake | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/cmake/BashCompletion.cmake b/cmake/BashCompletion.cmake
index a38dabda..49ef0a5d 100644
--- a/cmake/BashCompletion.cmake
+++ b/cmake/BashCompletion.cmake
@@ -1,4 +1,5 @@
 set(FORCE_BASH_COMPLETION_INSTALLATION FALSE CACHE BOOL "Force bash completion installation")
+set(DISABLE_BASH_COMPLETION FALSE CACHE BOOL "Disable bash completion installation")
 
 if(FORCE_BASH_COMPLETION_INSTALLATION AND "${BASH_COMPLETION_COMPLETIONSDIR}" STREQUAL "")
     set(BASH_COMPLETION_COMPLETIONSDIR "/share/bash-completion/completions")
@@ -29,6 +30,10 @@ if(PKG_CONFIG_FOUND OR FORCE_BASH_COMPLETION_INSTALLATION)
     if(NOT BASH_COMPLETION_UNAVAILABLE OR FORCE_BASH_COMPLETION_INSTALLATION)
 	set(BASH_COMPLETION_COMPLETIONSDIR "${CMAKE_INSTALL_PREFIX}${BASH_COMPLETION_COMPLETIONSDIR}" CACHE PATH "Bash completion installation directory" FORCE)
         mark_as_advanced(BASH_COMPLETION_COMPLETIONSDIR)
-        set(BASH_COMPLETION_FOUND TRUE)
+        if(NOT DISABLE_BASH_COMPLETION)
+            set(BASH_COMPLETION_FOUND TRUE)
+        else()
+            set(BASH_COMPLETION_FOUND FALSE)
+        endif()
     endif()
 endif()
-- 
2.13.6

